hashtagsv2/hashtags/templates/hashtags/docs.html

```diff
@@ -2,17 +2,17 @@
     <img src="http://31.media.tumblr.com/89e0dadf1eb2059d6811134ecd75e92c/tumblr_inline_nkq9fdb40Z1ql4e1e.png">
-    <p>Learn more <a href="http://blog.hatnote.com/post/112756032432/the-humble-hashtag-now-on-wikipedia">on the Hatnote blog</a>.</p>
-    <h3>What is a hashtag?</h3>
-    <p>We follow <a href="https://github.com/twitter/twitter-text/blob/master/conformance/autolink.yml#L128">Twitter's specification</a> (mostly) for hashtags: a hash mark (#) followed by one or more alphanumeric characters.</p>
-    <p>We do, however, ignore any hashtags which <i>only</i> contain numbers, as these usually denote someone counting rather than a use of a hashtag we would be interested in tracking.</p>
-    <h3><a name="download"></a>Downloading results</h3>
-    <p>You can download CSV results for a hashtag at <code>http://hashtags.wmflabs.org/csv/?query&lt;hashtag&gt;</code>. You can also optionally provide a <code>project</code> parameter to limit your results to one Wikimedia project (e.g. <code>fr.wikisource.org</code>, and <code>startdate</code> and/or <code>enddate</code> parameters to limit your search by date (date must be formatted as YYYY-MM-DD).</p>
-    <h3>Is this a new version of the tool?</h3>
-    <p>Yes! Partly. The hashtags tool was originally created by Stephen LaPorte and Mahmoud Hashemi in 2016. It was hosted on Toolforge and was used by Wikimedia editors and program leaders from all around the Wikimedia movement. By 2018, however, the tool had outgrown its setup and hosting - a database with over 8 million hashtag entries had accumulated and the tool was sluggish, often unresponsive, and was <a href="https://phabricator.wikimedia.org/T188205">becoming a drain on other Toolforge tools' databases</a>.</p>
-    <p>In August 2018, Sam Walton started work revitalising the tool, switching the backend from Flask to Django and setting it up on a dedicated Cloud VPS Project with dedicated database resources. This version of the tool also monitors the <a href="https://wikitech.wikimedia.org/wiki/EventStreams">recentchanges MediaWiki EventStream</a> directly instead of periodically reading from the recentchanges database table. This means it now updates in near-realtime and covers all Wikimedia projects (except Wikidata, for now). The tool no longer tracks bot-flagged edits to avoid filling up with edits that can be easily tracked in other ways (e.g. InternetArchiveBot).
+    <p>{% trans "Learn more " %}<a href="http://blog.hatnote.com/post/112756032432/the-humble-hashtag-now-on-wikipedia">{% trans "on the Hatnote blog" %}</a>.</p>
+    <h3>{% trans "What is a hashtag?" %}</h3>
+    <p>{% trans "We follow " %}<a href="https://github.com/twitter/twitter-text/blob/master/conformance/autolink.yml#L128">{% trans "Twitter's specification" %}</a> {% trans "(mostly) for hashtags: a hash mark (#) followed by one or more alphanumeric characters." %}</p>
```

Comment:
On this line and others you break up sentences into multiple `trans` blocks. While this removes potentially confusing HTML, it makes it almost impossible for a translator to translate the full sentence - they'll be presented with each section at a time (e.g. "We follow", and then "Twitter's specification", and then "(mostly) for hashtags...").

Translators need the full sentence so that the result makes sense. It's ok to include some HTML/Jinja markup inside the `trans` block if it keeps the whole sentence together.